packages/runtime/container-runtime/src/blobAggregationStorage.ts

```diff
@@ -0,0 +1,388 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+ import {
+     IDocumentStorageService,
+     ISummaryContext,
+} from "@fluidframework/driver-definitions";
+ import {
+    ISnapshotTree,
+    ISummaryHandle,
+    ISummaryTree,
+    IVersion,
+    SummaryType,
+    ITree,
+} from "@fluidframework/protocol-definitions";
+import {
+    assert,
+    bufferToString,
+    stringToBuffer,
+    unreachableCase,
+    fromUtf8ToBase64,
+    Uint8ArrayToString,
+ } from "@fluidframework/common-utils";
+import { ITelemetryLogger } from "@fluidframework/common-definitions";
+
+// Gate that when flipped, instructs to compress small blobs.
+// We have to first ship with this gate off, such that we can get to saturation bits
+// that can understand compressed format. And only after that flip it.
+function gatesAllowPacking() {
+    // Leave override for testing purposes
+    if (typeof localStorage === "object" && localStorage !== null && localStorage.FluidAggregateBlobs === "1") {
+        return true;
+    }
```

Comment:
i don't like this here. can we use a loader option, we can then se the loader option based on localStorage.

Reply:
This is strictly for testing purposes. I.e. be able to overwrite setting locally in dev tools.
It is also temporarily here, for the moment when client bits containing ability to read aggregated blobs are propagating, and before we pull the plug to start writing. Once writing is enabled in prod, that code would be deleted.
I'm not sure I see a good reason to expose it on loader / make code changes in consumers. If anything, it should be an option in SPO driver factory, not loader.